contrib/browndog/PEcAn#DALEC_convert.R

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/Rscript
-#PEcAn
-#data
-#pecan.zip
-#dalec
+# PEcAn
```

Comment:
@robkooper   I am not sure if this addiction space will break browndog code to parse the input & output file format.